contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveScan.java

```diff
@@ -87,6 +87,9 @@
   @JsonProperty("columns")
   public List<SchemaPath> columns;
 
+  @JsonProperty("numPartitions")
+  public final int numPartitions;
```

Comment:
I couldn't find documentation in hive explicitly stating this. However based on some default parameters for dynamic partitions the int_max seems large enough.